packages/core/src/App/Containers/Layout/header/default-header.jsx

```diff
@@ -14,6 +15,18 @@ import { clientNotifications } from 'Stores/Helpers/client-notifications';
 import ToggleMenuDrawer from 'App/Components/Layout/Header/toggle-menu-drawer.jsx';
 import { AccountsInfoLoader } from 'App/Components/Layout/Header/Components/Preloader';
 
+const TryBeta = () => {
+    const history = useHistory();
+    return (
+        <div className='header__try-beta' onClick={() => history.push(routes.dashboard)}>
```

Comment:
shouldn't we navigate to `myapps.deriv.com`?

Reply:
yes, you are right, my mistake👍